flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/ByteBufUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.netty;
+
+import org.apache.flink.shaded.netty4.io.netty.buffer.ByteBuf;
+
+import javax.annotation.Nullable;
+
+/**
+ * Utility routines to process Netty ByteBuf.
+ */
+public class ByteBufUtils {
+
+	/**
+	 * Accumulates data from the source buffer to the target buffer. If no data has been accumulated yet and the source
+	 * buffer have enough data, source buffer will be returned directly. Otherwise, data will be copied into the target
+	 * buffer. If the size of data copied after this operation has exceed the expected total size, the target buffer
+	 * will be returned for read, otherwise null will be return to indicate more data is required.
+	 *
+	 * @param target The target buffer.
+	 * @param src The source buffer.
+	 * @param totalSize The total length to accumulate.
+	 * @param accumulatedSize The size of data accumulated so far.
+	 *
+	 * @return The ByteBuf containing accumulated data. If not enough data has been accumulated,
+	 * 		null will be returned.
+	 */
+	@Nullable
+	public static ByteBuf accumulate(ByteBuf target, ByteBuf src, int totalSize, int accumulatedSize) {
+		// If the accumulation buffer is empty and src has enough bytes,
```

Comment:
we can remove this comment because it is already mentioned in above method javadoc

Reply:
Removed the duplicated comment.